stable/xray/templates/xray-setup-conf.yaml

```diff
@@ -15,11 +15,20 @@ data:
     SCRIPTS_DIR=/scripts
     XRAYCONFIGPATH={{ .Values.common.xrayConfigPath }}
     XRAY_CONFIG_DIR=${XRAYCONFIGPATH}/config
+    XRAY_TLS_DIR=${XRAYCONFIGPATH}/tls
     XRAY_CONFIG_FILE=${XRAY_CONFIG_DIR}/xray_config.yaml
 
     echo "Creating directories"
     mkdir -pv ${XRAY_CONFIG_DIR}
 
+    # Set Postgres TLS certs
+    {{- if and .Values.global.postgresql.url .Values.global.postgresql.tlsSecret (not .Values.postgresql.enabled) }}
```

Comment:
Let's add a check that the `postgresql.url` contains the `sslkey=` and `sslrootcert=`, so we validate users actually put this path of the url and didn't forget. WDYT?

Reply:
not sure how to make not very complicated way, for now I improved examples how to set it up.